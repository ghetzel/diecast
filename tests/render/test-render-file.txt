type={{ underscore $.type }}
{{ range $item := $.items -}}
- {{ $item.id }}
{{ end -}}
debug={{ $.debug }}