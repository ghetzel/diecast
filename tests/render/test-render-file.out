Incorrect Usage.

NAME:
   diecast-linux-amd64 render - Render the given file as a template.

USAGE:
   diecast-linux-amd64 render [command options] [arguments...]

OPTIONS:
   --data, -d [--data option --data option]	Specify a data to include in the template input as a "[nested.]key=value" string.
   
